input/FluxCorrect/inclined_contact_ppm.incl

```diff
@@ -0,0 +1,94 @@
+ Boundary {
+     type = "periodic";
+ }
+
+
+ Domain {
+      lower = [0.0, 0.0, 0.0];
+      upper = [3.0, 1.5, 1.5];
+ }
+
+ Field {
+      list = ["density", "pressure", "velocity_x", "velocity_y", "velocity_z",
+	      "total_energy", "internal_energy"];
+
+      gamma = 1.6666666666666667;
+      ghost_depth = 4; # needs to be 4 for flux corrections
```

Comment:
PPM uses third order interpolation (I always mix this up myself). According to the Enzo Paper, it requires 3 ghost zones. We need to round up to 4 zones since AMR requires an even number of ghost zones.